sql/report_util_functions.sql

```diff
@@ -406,25 +404,24 @@ DELIMITER ;
 
 -- patientHasScheduledAnARTAppointmentDuringReportingPeriod
 
-DROP FUNCTION IF EXISTS patientHasScheduledAnARTAppointmentDuringReportingPeriod;
+DROP FUNCTION IF EXISTS patientHasScheduledAnARTAppointment;
 
 DELIMITER $$
-CREATE FUNCTION patientHasScheduledAnARTAppointmentDuringReportingPeriod(
+CREATE FUNCTION patientHasScheduledAnARTAppointment(
     p_patientId INT(11),
     p_startDate DATE,
-    p_endDate DATE) RETURNS TINYINT(1)
+    p_endDate DATE,
+    p_monthOffset INT(11)) RETURNS TINYINT(1),
```

Comment:
@IipperMangesh please remove the comma at the end (it is causing a syntax error that prevents the SQL to be compiled)